azext_iot/tests/digitaltwins/test_dt_import_jobs_int.py

```diff
@@ -0,0 +1,177 @@
+            if show_valid_import_job_output["status"] == "succeeded" and show_invalid_import_job_output["status"] == "failed":
+                assert(show_invalid_import_job_output["error"]["error"]["code"] == "DTDLParsingError")
+                assert(show_valid_import_job_output["error"] is None)
+                break
+            sleep(30)
+
+        list_models_output = self.cmd(
+            "dt model list -n {} -g {} --definition".format(instance_name, self.rg)
+        ).get_output_in_json()
+        assert len(list_models_output) == 10
+        for model in list_models_output:
+            assert model["id"]
+            assert model["model"]
+
+        twin_query_result = self.cmd(
+            "dt twin query -n {} -q 'select * from digitaltwins'".format(instance_name)
+        ).get_output_in_json()
+        assert len(twin_query_result["result"]) == 10
+
+        twin_ids = [twin["$dtId"] for twin in twin_query_result["result"]]
+        relationship = "has"
+        for twin_id in twin_ids:
+            twin_relationship_list_result = self.cmd(
+                "dt twin relationship list -n {} -g {} --twin-id {} --relationship {}".format(
+                    instance_name, self.rg, twin_id, relationship
+                )
+            ).get_output_in_json()
+            assert len(twin_relationship_list_result) == 1
+
+        # Delete import jobs
+        self.cmd(
+            "dt job import delete -n '{}' -g '{}' -j '{}' -y".format(instance_name, self.rg, valid_import_job_id)
+        )
+        self.cmd(
+            "dt job import delete -n '{}' -g '{}' -j '{}' -y".format(instance_name, self.rg, invalid_import_job_id)
+        )
+
+        # Delete output files from storage container
+        if self.cmd(
+            "storage blob exists --connection-string '{}' -c '{}' -n '{}'".format(
+                self.storage_cstring, self.storage_container, valid_import_job_output_filename
+            )
+        ).get_output_in_json()["exists"]:
+            self.cmd(
+                "storage blob delete --connection-string '{}' -c '{}' -n '{}'".format(
+                    self.storage_cstring, self.storage_container, valid_import_job_output_filename
+                )
+            )
+
+        if self.cmd(
+            "storage blob exists --connection-string '{}' -c '{}' -n '{}'".format(
+                self.storage_cstring, self.storage_container, invalid_import_job_output_filename
+            )
+        ).get_output_in_json()["exists"]:
+            self.cmd(
+                "storage blob delete --connection-string '{}' -c '{}' -n '{}'".format(
+                    self.storage_cstring, self.storage_container, invalid_import_job_output_filename
+                )
+            )
+
+
+def assert_import_job_creation(create_import_job_output):
+    assert(create_import_job_output is not None)
+    assert(create_import_job_output["error"] is None)
+    assert(create_import_job_output["status"] == "notstarted")
```

Comment:
can you start a job? what happens then?

Reply:
As a user, you can only create a job. The API handles starting the job.